spec/views/curation_concerns/base/_items.html.erb_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe 'curation_concerns/base/items', type: :view do
+  let(:ability) { double }
+  let(:presenter) { Sufia::WorkShowPresenter.new(solr_doc, ability) }
+  let(:file_set) { Sufia::FileSetPresenter.new(solr_doc_file, ability) }
+  let(:member) { Sufia::WorkShowPresenter.new(solr_doc_work, ability) }
+  let(:solr_doc) { double(id: '123', human_readable_type: 'Work') }
+  let(:solr_doc_file) do
+    SolrDocument.new(
+      FactoryGirl.build(:file_set).to_solr.merge(
+        id: "file",
+        title_tesim: "Child File",
+        label_tesim: "Child File"
+      )
+    )
+  end
+  let(:solr_doc_work) do
+    SolrDocument.new(
+      FactoryGirl.build(:generic_work).to_solr.merge(
+        id: "work",
+        title_tesim: "Child Work"
+      )
+    )
+  end
+  let(:blacklight_config) { CatalogController.new.blacklight_config }
+  let(:blacklight_configuration_context) do
+    Blacklight::Configuration::Context.new(controller)
+  end
+
+  context "when children are present" do
+    before do
+      stub_template 'curation_concerns/base/_actions.html.erb' => 'Actions'
+      allow(presenter).to receive(:file_set_presenters).and_return([file_set])
```

Comment:
I guess not. The test wouldn't go red without it but i guess it makes sense to remove now.